### ISSUE TITLE
[request] Enable support for backblaze

### ISSUE THREAD

**Issue by MayurDuduka** [skyplane-project/skyplane#805]
**Please describe problem to be solved**  
(Replace This Text: Please present a concise description of the problem to be addressed by this feature request. Please be clear what parts of the problem are considered to be in-scope and out-of-scope.)

**(Optional): Suggest a solution**  
(Replace This Text: A concise description of your preferred solution. Things to address include:
* Details of the technical implementation
* Tradeoffs made in design decisions
* Caveats and considerations for the future)
* https://www.backblaze.com/

**Comment by sarahwooders** [comment]
@MayurDuduka thanks for the suggestion! We are currently working on adding support for generic S3 endpoints in #800, so we should be able to add Blackblaze support once that is done. Would you be interested in contributing the integration?
